tests/Exizent.CaseManagement.Client.IntegrationTests/UpdatingEstateItemNotes.cs

```diff
@@ -0,0 +1,233 @@
+        using var scope = serviceProvider.CreateScope();
+        var client = scope.ServiceProvider.GetRequiredService<ICaseManagementApiClient>();
+
+        Func<Task> action = async () => await client.UpdateEstateItemNotes(caseId, estateItemId, "some notes");
+
+        await action.Should().ThrowAsync<HttpRequestException>()
+            .WithMessage("Response status code does not indicate success: 404 (Not Found).");
+    }
+
+    [Fact]
+    public async Task ShouldThrow404NotFoundExceptionIfDocumentIsNotFound()
+    {
+        var clientId = Guid.NewGuid().ToString();
+        var clientSecret = Guid.NewGuid().ToString();
+
+        var caseId = Guid.NewGuid();
+        var estateItemId = Guid.NewGuid();
+
+        using var authApiServer = WireMockServer.Start();
+        authApiServer
+            .Given(Request.Create().WithPath("/oauth/token")
+                .WithBody(new JsonPartialMatcher(
+                    $@"{{ ""grant_type"": ""client_credentials"", ""client_id"": ""{clientId}"", ""client_secret"": ""{clientSecret}"", ""scope"": ""{ExizentScopes.All}"", ""audience"": ""https://resources.exizent.com"" }}"))
+                .UsingPost())
+            .RespondWith(Response.Create()
+                .WithBody("{ \"access_token\": \"123456\", \"expires_in\": 3600, \"token_type\": \"Bearer\" }")
+            );
+
+        using var casesApiServer = WireMockServer.Start();
+        casesApiServer.Given(
+                Request.Create().WithPath($"/cases/{caseId}/estateitems/{estateItemId}/notes").UsingPut()
+                    .WithBody("some notes")
+                    .WithHeader("User-Agent", "My browser")
+            )
+            .RespondWith(Response.Create()
+                .WithHeader("Authorization", "Bearer 123456")
+                .WithStatusCode(HttpStatusCode.NotFound)
+            );
+
+        var baseUri = casesApiServer.Url;
+        var baseAuthorizationUri = authApiServer.Url;
+
+        var serviceContainer = new ServiceCollection();
+        serviceContainer.AddExizentCaseManagementClient(
+            clientId,
+            clientSecret,
+            settings =>
+            {
+                settings.BaseUri = new Uri(baseUri);
+                settings.BaseAuthorizationUri = new Uri(baseAuthorizationUri);
+                settings.Scope = ExizentScopes.All;
+                settings.UserAgent = "My browser";
+            }
+        );
+
+        await using var serviceProvider = serviceContainer.BuildServiceProvider();
+        using var scope = serviceProvider.CreateScope();
+        var client = scope.ServiceProvider.GetRequiredService<ICaseManagementApiClient>();
+
+        Func<Task> action = async () => await client.UpdateEstateItemNotes(caseId, estateItemId, "some notes");
+
+        await action.Should().ThrowAsync<HttpRequestException>()
+            .WithMessage("Response status code does not indicate success: 404 (Not Found).");
+    }
+
```

Comment:
This doesn't actually test if wrong body sent. This mans that the TEST could not find an appropriate response.  On line 198 onwards, that given says, 'if I see a request with this URL and this body then return the response 'No content'.  Because you send in 'some notes' the test doesn't find any response to give back and therefore the 'Not found' is coming from the test framework rather than the mocked api.

Reply:
This test can therefore be removed